docs/onboard-to-seed.md

```diff
@@ -0,0 +1,50 @@
+# Onboard to SEED
+
+Currently Public Officers can onboard their device to SEED by logging into the TechPass portal using non-SE GSIB machine.
+
+<div class="warn">
+ <ul>
+ <li>SEED is applicable only for <b>non-GSIB device</b>.</li>
+ <li>If you are a vendor or contractor who needs to be onboarded to SEED, contact your project manager or reporting officer.</li>
+ <li>Click <a href="https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices">here</a> for more information about SEED.</li>
+ </ul>
+</div>
+
+_To onboard your device to SEED:_
+
+1. Go to [TechPass portal](https://portal.techpass.gov.sg) and click **Login with TechPass**.
+
+<kbd>![log in](assets/images/onboarding/po-non-se/log-in-with-techpass.png ':size=90%')</kbd>
+
+2. Choose your TechPass account.
+
+<kbd>![choose-account](assets/images/onboarding/po-non-se/choose-account.png)</kbd>
+
+?> If you are a public officer and not logged into your WOG account, you will be prompted to log in to it first before proceeding to TechPass account.
```

Comment:
this statement is abit confusing. is this the sign in prompt that appears sometimes when your wog browser session is timed out?

if yes, then I think can move this before step 1. because we will likely see it before we hit our landing page.
and can phrase it this way "You may be prompted to sign in with your WOG account before you hit TechPass portal landing page. This happens when your browser session timed out."

Reply:
hm i thought we will first need to select the account to login? e.g. if i select a vendor account then i won't be prompted with WoG sign-in right? This is actually copied from the 'Edit Profile' doc

Reply:
so by default when a po sign into their gsib device, a browser session is established at the same time. so normally, po didn't need to select any account or enter password to sign into techpass.
only occasionally when that browser session timeouts, then does the po needs to sign in.

as for vendors/contractors, they don't have gsib device usually. so they can't sign into techpass portal.

in that case,see ** for suggest changes. then delete this ?> line. @SenthilKumarSivakami you ok with it?

Reply:
deleted this note and added the following before the screenshot

?> Note If you are not logged into your WOG account, you will be prompted to log in to that before signing in to your TechPass account.